### DISCUSSION TITLE
Is it possible to use a custom cert?

### DISCUSSION THREAD

**Discussion by aqeelat** [vercel/community#760] [Help] [answered]
The 90 days provided by LE is very short. Is there a way to use a custom cert?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @AqeelAT. An Enterprise plan is required to upload a custom certificate. The majority of cases do not need a custom cert, but if you can share more details about your use case I will figure out if it's something we can already handle or might do in the future.

**Reply by aqeelat** [reply]
Thank you for your rapid response.

The issue we're facing is having to verify our domain with Apple for use of ApplePay. They require us to upload a static file 30 days before the https cert expires, which means we'll have to do it very frequently. By providing a longer custom cert, this will be a yearly problem instead of a quarterly problem.

Apple claim they have auto-reverification but so far it hasn't been working for us. (Maybe they require an OV cert?)
https://developer.apple.com/forums/thread/663425
https://developer.apple.com/forums/thread/701573

Do you have clients who support ApplePay and are not having the same issue?
